app/assets/javascripts/leaflet.layers.js

```diff
@@ -31,6 +31,9 @@ L.OSM.layers = function (options) {
         var miniMap = L.map(mapContainer[0], { attributionControl: false, zoomControl: false, keyboard: false })
           .addLayer(new layer.constructor(layer.options));
 
+        if (layer.options.schemeClass) miniMap.getPane("tilePane").classList.add(layer.options.schemeClass);
+        miniMap.getPane("tilePane").style.setProperty("--dark-mode-map-filter", layer.options.filter || "none");
```

Comment:
If the dark flag/class should have special treatment, should thunderforest-cycle get a special `nofilter` class as the first layer where the default from the current proposal https://github.com/openstreetmap/openstreetmap-website/issues/5328#issuecomment-2539051340 is explicitly asked for (https://github.com/openstreetmap/openstreetmap-website/issues/5395#issuecomment-2538451264)? And should further layers also get a class each?

Do you think it is favorable to spread the map layer filter definitions that much into CSS as well? I think having the way the layers from leaflet-osm are used in a single location including filters adds to the maintainability.

Reply:
Since the opinions in #5328 varied wildly, I wanted to have an easy flag for other devs that an alternate tileset has been loaded, since some might like a different styling like I already mentioned in https://github.com/openstreetmap/openstreetmap-website/pull/5396#issue-2736705476.
You are correct in that it doesn't do anything on the unmodified site.